FakeTrello/Models/Card.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Web;
+
+namespace FakeTrello.Models
+{
+    public class Card
+    {
+        [Key]
+        public int CardId { get; set; }
+
+        public string Title { get; set; }
+
+        public string Description { get; set; }
+
```

Comment:
It's not necessary for Entity Framework to have your navigation properties go both ways (Card having a List and List having a List<Card>), but it's helpful for when you aren't sure how your application will need to use your models. As you have it now, if you are ever working with a Card in your application, you can't easily access the List it belongs to. Like I said, not necessary here, but it's an important consideration in your models going forward! 😄